flaky/_flaky_plugin.py

```diff
@@ -233,7 +233,11 @@ def _add_flaky_report(self, stream):
             `file`
         """
         stream.write('===Flaky Test Report===\n\n')
-        stream.write(self._stream.getvalue())
+        value = self._stream.getvalue()
+        try:
+            stream.write(value)
+        except UnicodeEncodeError:
+            stream.write(ensure_byte_string(value))
```

Comment:
These five lines can be compressed into ``stream.write(ensure_byte_string(self._stream.getvalue()))``.